code/update.lua

```diff
@@ -67,8 +69,16 @@ while next(requests) do
   local ok, err = handleHttpEvent(os.pullEvent())
   if not ok then
     printError(err)
+    if fs.exists("codeOld/") then
+      fs.move("code/","codeOld/")
+      fs.move("code.lua","code.lua.old")
+      print("Recovred old files")
+    end
     break
   end
 end
-
+if fs.exists("codeOld/") then
+  fs.delete("code/","codeOld/")
+  fs.delete("code.lua","code.lua.old")
+end
```

Comment:
```suggestion

if fs.exists("/code_old") then
  fs.delete("/code_old")
end

```

Give this `if` some space :D

Reply:
Yeah i was maybe too quick with my edits. I dont know how i missed the "then" statement, i guess that's why we have code reviews haha. I would definitely agree that proper argument handling is better in the future.

Anyways your edits look good. I have never really gotten to work properly with someone on CC:T projects before, i hope i can help out more in the future :D